### ISSUE TITLE
Proof for loss function

### ISSUE THREAD

**Issue by ZhouCX117** [dougbrion/pytorch-classification-uncertainty#12]
Hi, thanks for your excellent work. But I can't understand the derived loss function, and the proof is provided in the supplementary material. So where can I download it?
